components/DynamicForm/Tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import MuiTooltip from '@mui/material/Tooltip'
+import HelpIcon from '@mui/icons-material/Help'
+
```

Comment:
For clarification, is this file to create pop up text?

Reply:
Yup! This component makes the like line of text with the info question mark that you can hover over for the tooltip